doc/tutorials/increase-uos-disk-size.rst

```diff
@@ -0,0 +1,51 @@
+.. _Increase UOS disk size:
+
+Increasing the User OS disk size
+################################
+
+This document builds on the :ref:`getting_started` and assumes you already have
+a system with ACRN installed and running correctly. The size of the pre-built
+Clear Linux User OS (UOS) virtual disk is typically of 8GB. This may not be
+sufficient and this guide explains in a few steps how to increase the size of
+that virtual disk.
+
+This document is largely inspired from Clear Linux's `Increase virtual disk size
+of a Clear Linux* OS image <https://clearlinux.org/documentation/clear-linux/
+guides/maintenance/increase-virtual-disk-size>`_ tutorial. The process can be
+broken down in three steps:
+
+1. Increase the virtual disk size
+#. Resize the ``rootfs`` partition
+#. Resize the filesystem
+
+Increase the virtual disk size
+******************************
+
+.. note::
+
+   This step **must** be performed while the User OS (UOS) is powered off.
+
+As an example, let us add 10GB of storage to our virtual disk image called
+``uos.img``.
+
+.. code-block:: none
+
+   $ qemu-img resize -f raw uos.img +10G
```

Comment:
Is the qemu-img tool guaranteed to be installed, or do we need to tell how to install it?

Reply:
Ah no, good catch. Assuming a Clear Linux dev system, it can be installed with `sudo swupd bundle-add clr-installer`. I will update this PR.